tmdb/season.go

```diff
@@ -34,7 +34,7 @@ func GetSeason(showID int, seasonNumber int, language string, seasonsCount int,
 		URL: fmt.Sprintf("/tv/%d/season/%d", showID, seasonNumber),
 		Params: napping.Params{
 			"api_key":                apiKey,
-			"append_to_response":     "credits,images,videos,external_ids,alternative_titles,translations,trailers",
+			"append_to_response":     "credits,images,videos,translations,external_ids",
```

Comment:
Why do we remove "trailers" and "alternative_titles" everywhere with this change?

Reply:
@elgatito 

1. i removed `alternative_titles` only from season/episode b/c as i mentioned in description they do not have it.

>alternative_titles (у серии/сезона их нет)

only movie/show has `alternative_titles`.

2. about `trailers`
>trailers это что-то старое только для фильмов, videos новое для всех типов.

so i replaced legacy property `trailers` (which was working only for movies) with `videos` (which works for all video types).